diff --git a/hello.java b/hello.java
index 706d0dc..369c66c 100644
--- a/hello.java
+++ b/hello.java
@@ -2,7 +2,6 @@ public class hello {
     public static void main(String[] args) {
         System.out.println("Hello, World!");
         System.out.println("Hello, World from branch feature1");
-        System.out.println("Hello, World from branch feature2");
         System.out.println("New feature1 change");
         System.out.println("New feature1 to be added here");
         System.out.println("New feature1 to be added here again");
